vignettes/enrich.Rmd

```diff
@@ -0,0 +1,79 @@
+
+---
+title: "Enrichment analysis workflow"
```

Comment:
Change it to something more scientific: eg Enrichment analysis of molecular QTL in genetic variants associated with complex traits